app/views/shipit/stacks/_header.html.erb

```diff
@@ -36,7 +36,7 @@
     </li>
     <% if stack.deploy_url.present? %>
       <li class="nav__list__item">
-        <%= link_to 'View website', stack.deploy_url, :target => '_blank', :class => 'deploy-url' %>
+        <%= sanitize link_to 'View website', stack.deploy_url, :target => '_blank', :class => 'deploy-url' %>
```

Comment:
Sanitize works on html tags, I tried to sanitize argument it didn't escape, 
http://stackoverflow.com/a/20413947